src/messaging/views.py

```diff
@@ -14,6 +15,11 @@
 
 # Create your views here.
 
+def getQuery(prof1, prof2):
```

Comment:
`getQuery` is a bit of an undescriptive name. let's also make this a static method, so it can work on the class reference.  you could use it like

`Thread.getProfileQuery(me, you)`